go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/onsi/gomega v1.10.1
 	github.com/openshift/api v0.0.0-20210331193751-3acddb19d360
 	github.com/openshift/build-machinery-go v0.0.0-20211213093930-7e33a7eb4ce3
-	github.com/openshift/library-go v0.0.0-20210406144447-d9cdfbd844ea
+	github.com/openshift/library-go v0.0.0-20210401181443-5cf97eb29077
```

Comment:
why we use a lower version of library-go?

Reply:
Fixed but it still confused me too: 

First the bug was fixed by [5cf97eb29077b69d28be0b350260d8c1f40a64d4](https://github.com/openshift/library-go/blob/5cf97eb29077b69d28be0b350260d8c1f40a64d4/pkg/operator/resource/resourceapply/generic.go) in 2 Apr.

But after that, the modification disappear until [80b76d711afbe5d39d8240da7a539f567bcdcf05](https://github.com/openshift/library-go/blob/80b76d711afbe5d39d8240da7a539f567bcdcf05/pkg/operator/resource/resourceapply/generic.go)

---

I have changed lib-go to `80b76d711afbe5d39d8240da7a539f567bcdcf05` which `20210407110438`